woocommerce-payments.php

```diff
@@ -24,6 +24,7 @@
 define( 'WCPAY_SUBSCRIPTIONS_ABSPATH', __DIR__ . '/vendor/woocommerce/subscriptions-core/' );
 
 require_once __DIR__ . '/vendor/autoload_packages.php';
+require_once __DIR__ . '/includes/class-wc-payments-features.php';
```

Comment:
Moved this here from `includes/class-wc-payments.php` so we could access `WC_Payments_Features` a little earlier (see below)